client/extensions/woocommerce/app/promotions/promotions-list-table.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import PromotionsListRow from './promotions-list-row';
+import Table from 'woocommerce/components/table';
+import TableRow from 'woocommerce/components/table/table-row';
+import TableItem from 'woocommerce/components/table/table-item';
+
+const PromotionsListTable = ( {
+	site,
+	promotions,
+	translate,
+} ) => {
+	const headings = [
+		translate( 'Promotion' ),
+		translate( 'Type' ),
```

Comment:
Probably want to an explanation / context for this for translators.

Reply:
context added. Thanks